### PR TITLE
Enable push notifications in Chat SDK

### PR DESCRIPTION
**Pull request by knvsl**
- Chat SDK supports `startPushNotifications` and `stopPushNotifications`
- ACS resource is linked with Azure Notification Hub. Chat app registers for push notifications, APNS delivers notifications for following events:
  - ChatMessageReceived
  - ChatMessageEdited
  - ChatMessageDeleted
  - ChatThreadCreated
  - ChatThreadDeleted
  - ChatThreadPropertiesUpdated
  - ChatThreadParticipantAdded
  - ChatThreadParticipantRemoved

**TODO: Move registration settings out of test env**

### CODE REVIEW

File: sdk/communication/AzureCommunicationChat/Source/Registrar/RegistrarSettings.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import Foundation
+
+internal enum RegistrarSettings {
+    static let endpoint = "https://edge.skype.net/registrar/testenv/v2/registrations"
```
Comment (knvsl) [inline sdk/communication/AzureCommunicationChat/Source/Registrar/RegistrarSettings.swift:30]: This need to be updated to not be testenv

**Comment by tjprescott** [comment]
@knvsl is work still being done on this PR? The last update was 3 months ago.

**Comment by angellan-msft** [comment]
> [@knvsl](/knvsl) [knvsl](/knvsl) added the [ Do Not Merge ](/Azure/azure-sdk-for-ios/labels/Do%20Not%20Merge) label [on Jun 29, 2021](#event-4955904782)

Hi Travis,
@knvsl is not with the team anymore. I'll take over the PR. Let me spend some time reading the codes and resolving the comments. 

Thanks!
Angel

**Comment by tjprescott** [comment]
I will close this PR for now. Feel free to reopen it when it's ready for review.

**Pull request closed without merging**